photoeditor/src/main/java/ja/burhanrashid52/photoeditor/shape/LineShape.kt

```diff
@@ -28,13 +40,64 @@ class LineShape : AbstractShape("LineShape") {
 
     private fun createLinePath(): Path {
         val path = Path()
+
+        if (pointerLocation == ArrowPointerLocation.START || pointerLocation == ArrowPointerLocation.BOTH) {
```

Comment:
Moving `BOTH` condition first to make to more readable.
`if (pointerLocation == ArrowPointerLocation.BOTH  || pointerLocation == ArrowPointerLocation.START)`